doc/source/api/box.rst

```diff
@@ -0,0 +1,6 @@
+lammpsio.box module
```

Comment:
For all these, I would probably not include `module` in the header. You might also consider if we just use the module name, like `box`.

Reply:
I tried them and think just using the module name looks the best!